userguide/install.rst

```diff
@@ -139,21 +141,19 @@ On Ubuntu 17.10, there are issues with flash_tool loading the shared library 'li
     && sudo dpkg -i /tmp/libpng12.deb \
     && rm /tmp/libpng12.deb
 
-You will also need to use `the latest version of the tool <https://spflashtool.com/download/SP_Flash_Tool_v5.1744_Linux.zip>`_.
-
-4. Extract the zip files
-5. Start the tool with ``sudo``
-6. Select the ``*Android_scatter.txt`` file from the archive you downloaded in the first step as the scatter-loading file
-7. Choose "Firmware Upgrade"
-8. Double-check you chose "Firmware Upgrade" and not "Download" or "Format All"
+5. Open a Terminal, enter the SP flash tool directory and run it with ``sudo ./flash_tool.sh``.
```

Comment:
actually, since you are working on this and you did just run it ... have you tried running it without sudo? is there anything we can give our users as to what kind of access it actually needs? maybe as long as usb port access rights are ok, no sudo is needed? did you try without sudo? (I feel so uncomfortable with telling our users to sudo some random tool gotten from some random website)

Reply:
I agree with you. without sudo the program opens but I haven't tried flashing since I'm using the phone I flashed with it and I don't have any other spare phones...